### PR TITLE
Docs contributions section revamp

### LINKED ISSUE #626: [Docs] Main media section reorganization
Due to a bigger amount of media content available, a revision for [this section](http://arrow-kt.io/docs/quickstart/blogs/) is needed to allow an easier navigation and information finding for any user.

### LINKED ISSUE #797: Blogs & Presentations
We have to redesign the page [Blogs & Presentations](http://arrow-kt.io/docs/quickstart/blogs/)

### PR DESCRIPTION
**Pull request by calvellido**
This PR revamp the media/contributions section giving it a more concise style:

screenshot_2019-01-22 arrow
 
_The clipped title and the bad setting of the Gitter button are due to the capturing technique, don't worry about them_

This closes #626
This closes #797

### CODE REVIEW

File: modules/docs/arrow-docs/docs/_config.yml [hunk 1]
```diff
@@ -1,10 +1,3 @@
-defaults:
-  -
```
Comment (raulraja) [inline modules/docs/arrow-docs/docs/_config.yml]: removing this breaks the api docs

File: modules/docs/arrow-docs/docs/_data/sidebar.yml [hunk 2]
```diff
@@ -0,0 +1,316 @@
+options:
```
Comment (raulraja) [inline modules/docs/arrow-docs/docs/_data/sidebar.yml]: where are the apidocs?

**Comment by raulraja** [comment]
Looks great but I'm missing content

File: modules/docs/arrow-docs/docs/_data/sidebar.yml (hunk 2 above)
Comment (calvellido) [reply modules/docs/arrow-docs/docs/_data/sidebar.yml]: Yep, didn't actually update the menu items, my bad 🤦

File: modules/docs/arrow-docs/docs/_config.yml (hunk 1 above)
Comment (calvellido) [reply modules/docs/arrow-docs/docs/_config.yml]: Indeed, will fix it in the next commit too, thanks for noticing it :+1:

**Comment by raulraja** [comment]
Awesome! thanks and outstanding work, can't wait to see this live!

**Pull request merged**